secondary_indexes_test.py

```diff
@@ -383,6 +388,7 @@ def test_failing_manual_rebuild_index(self):
         assert before_files == after_files
         assert_none(session, """SELECT * FROM system."IndexInfo" WHERE table_name='k'""")
         assert_one(session, "SELECT * FROM k.t WHERE v = 1", [0, 1])
+        assert 2 == len(node.grep_log('became not-writable'))
```

Comment:
As before, it would be useful to verify that it has indeed become not-writable:
```suggestion
        assert 2 == len(node.grep_log('became not-writable'))
        
        # Insert a row, that row shouldn't be visible by the index because it's not writable
        session.execute("INSERT INTO k.t(k, v) VALUES (4, 1)")
        assert_one(session, "SELECT * FROM k.t WHERE v = 1", [0, 1])
```
This write should be visible as soon as we successfully rebuild the index, as we do a few lines later.